public/index.html

```diff
@@ -4,6 +4,24 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#1400ff">
+    <!-- Primary Meta Tags -->
+    <title>GoGetta for NYC - free food, clothing, shelter and social services</title>
+    <meta name="title" content="GoGetta for NYC - free food, clothing, shelter and social services">
+    <meta name="description" content="If you need free food, clothing, personal care items or shelter in New York City during COVID-19, search GoGetta: the best place to get up to date, local, community-verified information.">
```

Comment:
Good question.

Perhaps this would make more sense?
> If you need free food, clothing, _personal care or shelter services_ in New York City during COVID-19, search GoGetta: the best place to get up to date, local, community-verified information.

Reply:
That does sound potentially more accurate to me, but we'll see what Adam says on Slack.